test/pathNotEq.js

```diff
@@ -10,22 +10,15 @@ describe('pathNotEq', function() {
     obj = { a: { b: 1 } };
   });
 
-  it('should curry', function() {
-    assert.isTrue(RA.pathNotEq(['a', 'b'], 'foo', obj));
-    assert.isTrue(RA.pathNotEq(['a', 'b'])('foo', obj));
-    assert.isTrue(RA.pathNotEq(['a', 'b'], 'foo')(obj));
-    assert.isTrue(RA.pathNotEq(['a', 'b'])('foo')(obj));
-  });
-
-  it('tests path value is not equal', function() {
+  it('should return true if given path values are not equal', function() {
     assert.isTrue(RA.pathNotEq(['a', 'b'], 'foo', obj));
   });
 
```

Comment:
Can we break this into `context/specify` ? THanks

```js
context('given path values are equal', function() {
  specify('should return false', function() {
  ...assertions...  
  });
});

Reply:
Yes, of course.